priam/src/main/java/com/netflix/priam/restore/RestoreTokenSelector.java

```diff
@@ -54,11 +62,52 @@ public RestoreTokenSelector(ITokenManager tokenManager, @Named("backup") IBackup
      *            Date for which the backups are available
      * @return Token as BigInteger
      */
-    public BigInteger getClosestToken(BigInteger tokenToSearch, Date startDate) {
+    public BigInteger getClosestToken(String tokenToSearch, Date startDate)
```

Comment:
The changes to this are purely to improve the error message when finding the closest token is invalid, because the snapshot was taken with multiple tokens.